doc/classes/AnimationPlayer.xml

```diff
@@ -81,6 +93,12 @@
 				Returns [code]true[/code] if an animation is currently playing (even if [member speed_scale] and/or [code]custom_speed[/code] are [code]0[/code]).
 			</description>
 		</method>
+		<method name="is_playing_section" qualifiers="const">
+			<return type="bool" />
+			<description>
+				Returns [code]true[/code] if an animation is currently playing with section.
```

Comment:
This sentence doesn't quite parse for me, what does "playing with section" mean?

Reply:
Oh indeed. Since sections can now be set even if they are not playing, the API name should be changed to `has_section()`.